feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -49,6 +49,16 @@ func EqualToDefault[T any](query ygnmi.SingletonQuery[T], val T) check.Validator
 	return check.Equal(query, val)
 }
 
+// checkSliceContains checks if given string s1 is found in string array s2.
+func checkSliceContains(s1 string, s2 []string) bool {
```

Comment:
Let's try to enhance the check package, instead of adding one time helpers here.

Reply:
removed this code and changes are committed.